drlineconfiguration.json

```diff
@@ -1389,24 +1533,30 @@
     "name": "PathwayCommons",
     "displayName": "PathwayCommons",
     "category": "UNK",
-    "uriLink": "http://www.pathwaycommons.org/search?type=Pathway&q=%s"
+    "uriLink": "http://www.pathwaycommons.org/search?type=Pathway&q=%id"
   },
   {
     "name": "Pharos",
     "displayName": "Pharos",
-    "category": "UNK",
-    "uriLink": "https://pharos.nih.gov/targets/%value"
+    "category": "OTHER",
+    "uriLink": "https://pharos.nih.gov/targets/%id"
   },
   {
     "name": "DrugCentral",
     "displayName": "DrugCentral",
-    "category": "UNK",
-    "uriLink": "http://drugcentral.org/target/%value"
+    "category": "CHEMISTRY",
+    "uriLink": "http://drugcentral.org/target/%id"
   },
   {
     "name": "ABCD",
     "displayName": "ABCD",
     "category": "PAM",
-    "uriLink": "https://web.expasy.org/cgi-bin/abcd/search_abcd.pl?input=%u"
+    "uriLink": "https://web.expasy.org/cgi-bin/abcd/search_abcd.pl?input=%id"
+  },
+  {

```

Comment:
The model data lacks this https://wwwdev.ebi.ac.uk/uniprot/api/uniprotkb/accession/E9PXF8 while it is in the current site: https://www.uniprot.org/uniprot/E9PXF8